activerecord/lib/active_record/associations/collection_association.rb

```diff
@@ -453,6 +409,12 @@ def null_scope?
         owner.new_record? && !foreign_key_present?
       end
 
+      def find_from_target?
```

Comment:
Yes, this method is called by collection proxy like `null_scope?`.
https://github.com/rails/rails/pull/26200/files#diff-bee622c17de67f292adf310f75288e25R1126